arch/arm/src/nrf52/nrf52_sdc.c

```diff
@@ -283,32 +265,8 @@ static void on_hci(void)
 
           len = sizeof(*hdr) + hdr->len;
 
-#ifdef CONFIG_DEBUG_WIRELESS_INFO
-          if (hdr->evt == BT_HCI_EVT_CMD_COMPLETE)
-            {
-              struct hci_evt_cmd_complete_s *cmd_complete =
-                  (struct hci_evt_cmd_complete_s *)
-                      (g_sdc_dev.msg_buffer + sizeof(*hdr));
-              uint8_t *status = (uint8_t *)cmd_complete + 1;
-
-              wlinfo("received CMD_COMPLETE from softdevice "
-                     "(opcode: %d, status: 0x%x)\n",
-                     cmd_complete->opcode, *status);
-            }
-          else
-            {
-              wlinfo("received HCI EVT from softdevice "
-                     "(evt: %d, len: %zu)\n", hdr->evt, len);
-            }
-#endif
-
-          outbuf = bt_buf_alloc(BT_EVT, NULL, BLUETOOTH_H4_HDRLEN);
-          bt_buf_extend(outbuf, len);
-
-          memcpy(outbuf->data, g_sdc_dev.msg_buffer, len);
-
-          bt_hci_receive(outbuf);
-
+          g_bt_driver->receive(&g_bt_driver, BT_EVT,
```

Comment:
Maybe we can add function like `bt_msg_reveive(&g_bt_driver, BT_EVT, g_sdc_dev.msg_buffer, len)` instead of directly calling callback function which is set by BT kernel driver.

Reply:
Done, I add a symbol define to instead of driver details,
https://github.com/apache/incubator-nuttx/pull/3754/files#diff-bafaf660e053e10be22390e6b4e0d76d2a60143c99508c845e21ece5999b5c2aR56-R57